umm-spec-lib/src/cmr/umm_spec/migration/version/granule.clj

```diff
@@ -203,3 +219,19 @@
       (assoc :MetadataSpecification {:URL "https://cdn.earthdata.nasa.gov/umm/granule/v1.6.2"
                                      :Name "UMM-G"
                                      :Version "1.6.2"})))
+
+(defmethod interface/migrate-umm-version [:granule "1.6.3" "1.6.2"]
+  [context g & _]
+  (-> g
+      (specify-metadata "1.6.2")
+      (update :RelatedUrls
+              (fn [related-url]
+                (into []
+                      (remove #(and (= "EXTENDED METADATA" (:Type %))
+                                    (some #{(:SubType related-url)} ["DMR++" "DMR++ MISSING DATA"]))
+                              related-url))))))
+
+(defmethod interface/migrate-umm-version [:granule "1.6.2" "1.6.3"]
+  [context g & _]
+  (-> g
+      (specify-metadata "1.6.3")))
```

Comment:
Can you please add unit tests for these functions in cmr.umm-spec.test.migration.version.granule.  It really helps when trouble shooting this part of code even versions later when something goes wrong.